src/appkit.cs

```diff
@@ -4037,6 +4156,15 @@ interface NSColorList : NSSecureCoding {
 		bool WriteToUrl ([NullAllowed] NSUrl url, [NullAllowed] out NSError error);
 	}
 
+	[Mac (10, 14)]
```

Comment:
`, onlyOn64: true` ?

Reply:
Likely true on all protocols. They weren't listed in the headers, so I had to get from the docs, which doesn't list 64-bit ness. Will add to be safe.